AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -23,46 +24,55 @@ extension ApplePayComponent {
 
         /// A list of fields that you need for a billing contact in order to process the transaction.
         /// Ignored on iOS 10.*.
-        public let requiredBillingContactFields: Set<PKContactField>
+        public var requiredBillingContactFields: Set<PKContactField> = []
 
         /// A list of fields that you need for a shipping contact in order to process the transaction.
         /// Ignored on iOS 10.*.
-        public let requiredShippingContactFields: Set<PKContactField>
+        public var requiredShippingContactFields: Set<PKContactField> = []
 
         /// A pre-populated billing address.
-        public let billingContact: PKContact?
+        public var billingContact: PKContact?
 
         /// The flag to toggle onboarding.
         /// If true, allow the shopper to add cards to Apple Pay if non exists yet.
         /// If false, then Apple Pay is disabled if the shopper doesn't have supported cards on Apple Pay wallet.
-        public let allowOnboarding: Bool
+        /// Default is false.
+        public var allowOnboarding: Bool = false
+
+        // Shipping methods supported by the merchant.
```

Comment:
Some of these properties have only 2 comment slashes `//` instead of `///` :)

Reply:
Nice catch!